lib/rutabaga/feature.rb

```diff
@@ -3,67 +3,22 @@
 
 module Rutabaga
   module Feature
-    def feature(feature_file = find_feature)
-
+    def feature(feature_file = nil)
+      RSpec.deprecate(
+        "Calling `feature` from an `it` block",
+        :message => "Calling `feature` from an `it` block " \
+                    "is deprecated.\nIt should now be called directly in the " \
+                    "`describe` block."
+      )
+
+      feature_file = Util.find_feature(feature_file || RSpec.current_example.description)
```

Comment:
it now works like this:

```ruby
feature "some_feature.feature" do
  step "first step" do
    ...
  end
end
```

Although you don't have to specify the feature file name. If you don't, it uses the spec files base file name as it did before.